benchmarking_sbi/config/config.yaml

```diff
@@ -3,6 +3,7 @@ compute_metrics: true
 
 # Hydra's overrides are used to change to different tasks and algorithms
 defaults:
+  - _self_
```

Comment:
We could make `_self_` the last element in the list (new behavior) because we are not using primary config overrides but fine to leave it as it is (https://hydra.cc/docs/next/upgrades/1.0_to_1.1/default_composition_order#primary-config-is-a-yaml-file)

Reply:
done [`1627d636b`](https://github.com/sbi-benchmark/results/pull/3/commits/1627d636b49926d68ff08844092ff9ee1b2a3871)